### PR TITLE
Make initial picker mode consistent and configuragle

### PR DESCRIPTION
**Pull request by bi0ha2ard**
current, next states were set based on the iteration order of the titles table, which seemed to have put "orgfiles" in current, but the picker was always set to `finders.headlines`.  the first toggle mode bind would therefore not work consistently.

This passes the default initial state  ("headlines") explicitly. Additionally it exposes it as an option for telescope:

```
:Telescope orgmode search_headings mode=orgfiles
```

will open the picker directly in the orgfiles mode.

### CODE REVIEW

File: lua/telescope-orgmode/actions.lua [hunk 1]
```diff
@@ -1,4 +1,3 @@
-require('telescope-orgmode.typehints')
```
Comment (bi0ha2ard) [inline lua/telescope-orgmode/actions.lua:1]: This is unrelated to my changes.
I removed this import because this file doesn't exist, and the plugin won't load.

**Comment by dtvillafana** [comment]
The typehints import is also breaking my install without manual workaround, please merge this.

**Comment by dtvillafana** [comment]
I created a separate PR for just the fix that we need: [PR](https://github.com/nvim-orgmode/telescope-orgmode.nvim/pull/23)

**Comment by seflue** [comment]
@bi0ha2ard Thanks for this fix, looks good to me. The bug actually already annoyed me, but I didn't found the time to look into it.

**Comment by seflue** [comment]
Btw, sorry for my late response. I will hopefully react faster in the future, now that I should get notification emails from GitHub. :sweat_smile:
